app/src/main/java/com/app/calculator/model/OperationModel.java

```diff
@@ -11,6 +11,9 @@ public double getOperation(String operator){
             case "+":
                 result = firstNumber + secondNumber;
                 break;
+            case "/":
```

Comment:
you could use the switch with the enum directly, it's easier to compare and you have to maintain just the enum in only one place